.github/workflows/smoke-test-ipv6.yaml

```diff
@@ -60,10 +67,13 @@ jobs:
             --set nodeinit.enabled=true \
             --set kubeProxyReplacement=strict \
             --set ipam.mode=kubernetes \
-            --set image.tag=latest \
-            --set image.pullPolicy=Never \
-            --set operator.image.tag=latest \
-            --set operator.image.pullPolicy=Never \
+            --set image.repository=quay.io/${{ github.repository_owner }}/cilium-ci \
+            --set image.tag=${{ steps.vars.outputs.tag }} \
+            --set image.pullPolicy=IfNotPresent \
+            --set operator.image.repository=quay.io/${{ github.repository_owner }}/operator \
+            --set operator.image.suffix=-ci \
+            --set operator.image.tag=${{ steps.vars.outputs.tag }} \
+            --set operator.image.pullPolicy=IfNotPresent \
```

Comment:
I guess we don't use hubble relay image in smoke tests, right?

Reply:
I don't think so since they were not previously specified. cc @gandro ?

Reply:
I'm not super familiar with the smoke tests. But unless `hubble.relay.enabled=true` is set (which it does not seem to be), the Relay image will not be used.